src/components/resources/assort/Guide.tsx

```diff
@@ -0,0 +1,31 @@
+import { Table, Text } from "@radix-ui/themes";
+
+const data = [
+  "How to quality check donated items meant for distribution;",
+  "How to sort donations into standard categories;",
+  "How to size clothing donations with a printable ASSORT sizing chart;",
+  "How to box and label the sorted donations and ready them for shipment.",
+];
+
+export const Guide = () => {
+  return (
+    <Table.Root size="3" className="bg-gray-50 w-full">
```

Comment:
**Fix:** We should only use tables for tabular data.  To keep things simple, can we stray from the design for this and use a <ul> instead?

Reply:
what should i use instead of table ??